cmd/kops/create_cluster.go

```diff
@@ -1203,3 +1248,34 @@ func setOverrides(overrides []string, cluster *api.Cluster, instanceGroups []*ap
 	}
 	return nil
 }
+
+func getZoneToSubnetProviderID(VPCID string, region string, subnetIDs []string) (map[string]string, error) {
+	res := make(map[string]string)
+	cloudTags := map[string]string{}
+	awsCloud, err := awsup.NewAWSCloud(region, cloudTags)
+	if err != nil {
+		return res, fmt.Errorf("error loading cloud: %v", err)
```

Comment:
Nit: I think it's typically better to `return nil, fmt.Errorf...` because then if you forget to check the error it is usually more obviously.